templates/articles/detail.html

```diff
@@ -1,23 +1,33 @@
 {% extends "base.html" %}
 
+{% block meta_title %}{{ article.title }}{% endblock %}
+{% block meta_description %}{{ article.summary }}{% endblock %}
+
 {% block body %}
-<article>
+<article lang={{ article.language }}>
   <h1>{{ article.title }}</h1>
 
+  <p class=reading-time>Reading time: {{ article.body|reading_time }} min.</p>
+
+  <p>Article in <span>{{ article.language }}</span></p>
+
   {% if article.has_image %}
   <img src="{{ article.image_url }}" alt="Image for {{ article.title }}">
   {% endif %}
 
   {% if article.summary %}
-  <p>
-    {{ article.summary }}
-  </p>
+  <p class=summary>{{ article.summary }}</p>
   {% endif %}
 
   {{ article.body|markdown }}
+
   <p>
-    Published on <time>{{ article.creation_date.date() }}</time>
+    Published on <time>{{ article.creation_date.date() }}</time> by {{ article.author }}
   </p>
+  <nav class=share>
+    <a href="https://twitter.com/share?url={{ request.url }}&text={{ article.title }}&via=cafebabel">Twitter</a>
```

Comment:
Shouldn't you urlencode the `request.url`?

Reply:
:+1: